warehouse/redshift/redshift.go

```diff
@@ -642,7 +642,7 @@ func (rs *HandleT) FetchSchema(warehouse warehouseutils.WarehouseT) (schema ware
 		return
 	}
 	if err == sql.ErrNoRows {
-		pkgLogger.Debugf("RS: No rows, while fetching schema from  destination:%v, query: %v", rs.Warehouse.Identifier,
+		pkgLogger.Infof("RS: No rows, while fetching schema from  destination:%v, query: %v", rs.Warehouse.Identifier,
```

Comment:
So the pkgLogger has child being set as `redshift`, we can do away with `RS:`, wdyt ?